e2e-tests-rust/Cargo.toml

```diff
@@ -10,7 +10,7 @@ categories = ["cryptography"]
 publish = false
 
 [dependencies]
-alloy-zksync = { git = "https://github.com/itegulov/alloy-zksync.git", rev = "692c5c2ca5defc88ac542f420d97c6756dadf9df" }
+alloy-zksync = { git = "https://github.com/popzxc/alloy-zksync.git", rev = "6a2e3cfe41cae9f92dc6672dbd0ce56ccc37e9b3" }
 alloy = { version = "0.6", features = ["full", "rlp", "serde", "sol-types", "getrandom", "provider-anvil-api", "json-rpc"] }
```

Comment:
Created an issue for upgrading alloy to match alloy-zksync https://github.com/matter-labs/anvil-zksync/issues/504

(nothing actionable here just noting)

Reply:
Will pick #504 once this is merged